Tests/UpstreamIntegrationTests/util/TestHelpers.swift

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright 2023 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+import Foundation
+
+/// Creates a valid interact URL using the provided location hint.
+/// - Parameters:
+///    - locationHint: The location hint String to use in the URL
+/// - Returns: The interact URL with location hint applied
+public func createUrl(with locationHint: String?) -> String {
```

Comment:
Since we moved this in a separate file I think it would make sense to call this createInteractUrl in case other helpers are added in the future